js/src/analysis/classicEditorData.js

```diff
@@ -0,0 +1,154 @@
+import debounce from "lodash/debounce";
+import { updateReplacementVariable } from "../redux/actions/snippetEditor";
+import fillReplacementValues from "../helpers/sendReplaceVarsToStore";
+import removeMarks from "yoastseo/js/markers/removeMarks";
+import tmceHelper, { tmceId } from "../wp-seo-tinymce";
+
+/**
+ * Represents the classic editor data.
+ */
+class ClassicEditorData {
+	/**
+	 * Sets the wp data, Yoast SEO refresh function and data object.
+	 *
+	 * @param {Function} refresh The YoastSEO refresh function.
+	 * @param {Object} store     The YoastSEO Redux store.
+	 * @returns {void}
+	 */
+	constructor( refresh, store ) {
+		this._refresh = refresh;
+		this.store = store;
```

Comment:
We are not sure what the intended codestyle/convention here should be. Either we give `this.store` and `this.data` underscores, or remove the underscore from `this._refresh`.
@atimmer If you have time, could you comment on what you prefer the code to look like going forward?

Reply:
Using `_` is a JavaScript convention for private properties. Because JavaScript doesn't have a way to specify properties as private the convention is to signal private things with an underscore.